### PR TITLE
Added Keywords for fetching replicaset in ns

### PR DESCRIPTION
**Pull request by hello2ray**
\<Remember to add meaningful title\>

\<Short description of the PR\>

Fixes #\<issue number\>

Before merge following needs to be applied:
- [x] At least one example testcase added in testcases/  -- done
- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs)
- [ ] PR entry added in CHANGELOG.md in **In progress** section -done
- [x] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI -- done
- [x] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI -- done 
```
fail_under = 82
```

### CODE REVIEW

**Comment by hello2ray** [comment]
Please complete the checklist.
Could you please help me here. What checklist I have to fill here. Do I need to add any files to generate docs

@m-wcislo
 
Collaborator
m-wcislo commented 1 hour ago • 
Yeah, so you have the list in the description of the PR. Let me know if you have any problems.

From what I see in CI coverage is now 82, so you can update the value.

@m-wcislo
m-wcislo reviewed 1 hour ago
testcases/replicaset/replicaset.robot

*** Test Cases ***
Replicaset test case example
    [Tags]    replicaset
 
@m-wcislo m-wcislo 1 hour ago • 
 Collaborator
For

All new testcases tagged as prerelease along other tags to exclude it from execution until released on PyPI

just add prerelease here next to [Tags] replicaset

 
@hello2ray hello2ray 1 hour ago Author
prerelease
I added tags prerelease and could you kindly help me in generate docs and coverage

@hello2ray	Reply…

**Comment by hello2ray** [comment]
@m-wcislo Can u pls check now

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Keyword for getiing role and rolebinding [#56](https://github.com/devopsspiral/KubeLibrary/pull/56) by [@satish-nubolab](https://github.com/satish-nubolab)
 - Bearer token authentication [#39](https://github.com/devopsspiral/KubeLibrary/pull/39) by [@m-wcislo](https://github.com/m-wcislo)
 - Keyoword for create and delete a cronjob [#71](https://github.com/devopsspiral/KubeLibrary/pull/71) by [@satish-nubolab](https://github.com/satish-nubolab)
+- Keywords for get replicaset in a namespace [#82](https://github.com/devopsspiral/KubeLibrary/pull/88) by [@hello2ray](https://github.com/hello2ray)
```
Comment (m-wcislo) [inline CHANGELOG.md:14]: should be #91

**Review by m-wcislo — COMMENTED** [review]
Please generate docs. You have the link in 
Library Documentation regenerated according to >>>[Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs)<<<

**Comment by hello2ray** [comment]
Pushed new code. Hence closing this

**Pull request closed without merging**